src/NzbDrone.Core/Parser/LanguageParser.cs

```diff
@@ -18,7 +18,9 @@ public static class LanguageParser
                                                                 RegexOptions.Compiled);
 
 
-        private static readonly Regex SubtitleLanguageRegex = new Regex(".+?[-_. ](?<iso_code>[a-z]{2,3})$", RegexOptions.Compiled | RegexOptions.IgnoreCase);
+        private static readonly Regex SubtitleLanguageRegex = new Regex(".+?[-_. ](?<iso_code>[a-z]{2,3})([-_. ]forced)?$", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
Might be sufficient to just check for `(?:[-_. ]|$)`. Feels a bit weird to explicitly check for forced while not actually doing anything with it. Might be better to return the language and whether it's forced or not.

Tests should be added for this as well.

Reply:
The current regex in `develop` only matches the language code at the end of the filename. I think that is a good method to avoid unwanted matches with movies like [*La Vie **en** rose*](https://www.imdb.com/title/tt0450188/). So with the `forced` in the regex, we make sure that the only thing that is allowed after a language code in the filename is the `forced` flag. Would you agree? Making it a non-capturing group, however, does makes sense of course.

As I'm fairly new to programming, I don't really know where to start with the tests for this. Could you point me to a file where I can add test cases?